# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/mol/startmol-typo.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mol-2003-06-20/src/startmol.in	2003-06-12 13:00:59.000000000 +0200
+++ mol-2003-06-20-devfs/src/startmol.in	2003-06-21 17:58:58.000000000 +0200
@@ -256,7 +256,7 @@
     [ "$LINUX" -a ! -f `getres -F console_video_prefs` ] && {
 	echo
 	echo "*************************************************************"
-	echo " No video modes has been configured. Please run 'molvconfig'"
+	echo " No video modes have been configured. Please run 'molvconfig'"
 	echo " as root to configure full screen video or disable console"
 	echo " video in /etc/mol/molrc.video."
 	echo "*************************************************************"
